deploy/kubernetes/images/stable-master/image.yaml

```diff
@@ -52,5 +52,5 @@ imageTag:
   # Don't change stable image without changing pdImagePlaceholder in
```

Comment:
Is this still true? Anything we need to change in test/k8s-integration/main.go?

Reply:
Yes, this is used in any test that builds & installs a driver, from https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/k8s-integration/main.go#L448

Reply:
eg our main presubmit :-)

Reply:
But we do not need to change it manually anymore in test/k8s-integration/main.go? I think this comment session needs to be updated.